source/user_guide.rst

```diff
@@ -20,115 +17,95 @@ Who should read this
 Getting Started
 ---------------
 
-To start, please follow dojot's installation guide in
-:doc:`install/compose_guide`. There you should find how to properly download a
-working copy of the components, how to minimally configure them, how to start
-them up and how to check whether they are working.
+To start, please follow dojot's installation guide in :doc:`install/compose_guide`. There you should find how to
+properly download a working copy of the components, how to minimally configure them, how to start them up and how to
+check whether they are working.
 
 dojot basics
 ------------
 
-Before using dojot, you should be familiar with some basic operations and
-concepts. They are very simple to understand and use, but without them, all
-operations might become obscure and senseless. It is advisable to checkout our
+Before using dojot, you should be familiar with some basic operations and concepts. They are very simple to understand
+and use, but without them, all operations might become obscure and senseless. It is advisable to checkout our
 :doc:`architecture` to get acquainted with all internal components.
 
-First of all, you should check out how to acccess dojot through its APIs, which
-is detailed in the next section. After that, there's an explanation of a few
-basic entities in dojot: devices, templates and flows (including a simple
+First of all, you should check out how to acccess dojot through its APIs, which is detailed in the next section. After
```

Comment:
I think this could be reversed:
First we explain the concepts. Then we show how to use them.
Otherwise I feel we'd end up throwing a lot of unintelligible information at the reader, just to explain it later on.

Curiously enough, it seems that the text is already in this order (the proposed one).

Reply:
Ok, I'll rewrite this.